workflow/applications/applications.py

```diff
@@ -167,14 +168,14 @@ def get_task_names(self) -> Dict[str, List[str]]:
         pass
 
     @staticmethod
-    def get_gfs_interval(gfs_cyc: int) -> str:
+    def get_gfs_interval(gfs_cyc: int) -> timedelta:
         """
         return interval in hours based on gfs_cyc
         """
 
-        gfs_internal_map = {'0': None, '1': '24:00:00', '2': '12:00:00', '4': '06:00:00'}
+        gfs_internal_map = {'1': '24H', '2': '12H', '4': '6H'}
```

Comment:
@aerorahul Any reason these keys are strings instead of just using ints and avoiding the cast in L179?

Reply:
They can be `int`.

Reply:
Also, looks like the `0` key (cycling only) was removed and needs to be added back in unless some if upstream stops this from being called.

Reply:
see the updated `gfs_cycled.py` line 253

Reply:
I can update 1, 2, 4 from strings to ints if that's desired. Thanks!